transactor/src/main/scala/io/mediachain/transactor/DatastoreRpcService.scala

```diff
@@ -0,0 +1,58 @@
+package io.mediachain.transactor
+
+// this is placed in io.mediachain.transactor to accompany the rest of the programs
+object DatastoreRpcService {
+  import io.mediachain.datastore.{DynamoDatastore, DatastoreService}
+  import io.mediachain.util.Properties
+  import io.grpc.Server
+  import scala.concurrent.ExecutionContext
+  import java.util.concurrent.Executors
+  import org.slf4j.LoggerFactory
+  
+  val logger = LoggerFactory.getLogger(RpcService.getClass)
+  
+  def main(args: Array[String]) {
+    if (args.length != 1) {
+      println("arguments: config-file")
+      sys.exit(1)
+    }
+    
+    val conf = Properties.load(args.head)
+    run(conf)
+  }
+  
+  def run(conf: Properties) {
+    val rpcPort = conf.getq("io.mediachain.datastore.rpc.port").toInt
+    val ctldir = conf.getq("io.mediachain.datastore.rpc.control")
+    val maxObjectSize = conf.getopt("io.mediachain.datastore.maxObjectSize") match {
+      case Some(str) => str.toInt
+      case None => 65536
```

Comment:
Not a huge fan of number literal defaults appearing in two places, someone will 100% forget to change one